tailwind.config.mjs

```diff
@@ -23,8 +23,11 @@ export default {
           dark: background.dark,
         },
         brand: {
+          black: "#1A1A1A",
+          "carbon-black": "#101010",
           white: "#ECEDEE",
           gray: "#7C7C7C",
+          "ash-gray": "#858585",
```

Comment:
The brand-black is good, as good as the brand-white that was already there. But let's try to remove carbon-black and ash-gray and use some built-in tailwind colors, like neutral-#